package/src/values/api.ts

```diff
@@ -6,5 +6,3 @@ declare global {
 
 const { SkiaValueApi } = global;
 export const ValueApi = SkiaValueApi;
-
-export const { createValue, createComputedValue } = ValueApi;
```

Comment:
When changing the imports / exports we might end up with a breaking change - if this is needed for the Mock I'm fine, but its not clear to me that this is the case?

Reply:
Shall we just deprecate it?

Reply:
No need :) I approved the PR :) :) This is such a small change.